### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by NomanRamzan**
Update mobile-version and add media queries to make it desktop-version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,137 +83,139 @@ <h2>I’am a Software Developer</h2>
       </div>
     </section>
```
Comment (BenMukebo) [inline index.html:84]: - [ ] Kindly make sure to position your My Recent Rorks section on the center of the page

||      Your       ||        Figma       ||
|-|----------------------|-| ----------------------|-|
|| ||||

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @NomanRamzan,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️  

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ] Please update your Readme, It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc

- I would also recommend to add the following: 

```
## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -535,3 +534,302 @@ div.item .langu {
...
+
+/* Desktop Version Start */
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    background: #3c3a39;
+    width: 100%;
+    height: 92px;
+  }
+
```
Comment (BenMukebo) [inline style.css:546]: - [ ] According to the project requirement you should make sure that the  Desktop version looks like the one in the design you chose, I can see that your website is not really responsive while switching from mobile to desktop looks 👇🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]

**Review by MrRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @NomanRamzan 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ implemented previous review requested changes
- ✔️ descriptive readme
- ✔️ descriptive PR

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**